src/components/Modal.tsx

```diff
@@ -28,7 +28,7 @@ export interface CustomModalProps {
 }
 
 // TODO Further adjustments gonna be done with LUN-312
```

Comment:
Optional: This comment is outdated and could be removed. Would be nice if you could remove it.

Reply:
done

Reply:
@charludo I think you didn't push it, did you?

Reply:
oops, sorry. It is now.